packages/core-components/src/layout/Sidebar/Bar.tsx

```diff
@@ -229,7 +229,9 @@ export const Sidebar = (props: SidebarProps) => {
   const { isMobile } = useContext(SidebarPinStateContext);
 
   return isMobile ? (
-    <MobileSidebar>{children}</MobileSidebar>
+    <nav>
+      <MobileSidebar>{children}</MobileSidebar>
+    </nav>
```

Comment:
This extra wrapping looks a bit scary! Perhaps we could update the relevant element within the `MobileSidebar` instead?

Reply:
I removed this additional nav and moved it to `MobileSidebar`. Unfortunately the tags there are material UI components, so I'm unable to update those.

Reply:
In many cases you can pass `component='nav'` to the MUI components, at least the `BottomNavigation` element should support that

Reply:
Oh that's great! Thanks for the tip